marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -353,7 +353,7 @@ describe('json scenario for table on browse documents page', () => {
     detailPage.getDocumentSource().should('contain', 'loadCustomersJSON');
     detailPage.getDocumentFileType().should('contain', 'json');
 
-    cy.waitUntil(() => detailPage.clickBackButton()); //Click on Back button to navigate back to the browse table view.
+    detailPage.clickBackButton(); //Click on Back button to navigate back to the browse table view.
```

Comment:
The test fails with this change as well. Can we try adding waitForSpinner @ line 349?

Reply:
Tests seems to be passed in the second run.